### ISSUE TITLE
[Bug] The engine was not killed due to the operation timeout

### ISSUE THREAD

**Issue by wForget** [apache/incubator-kyuubi#2450]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

Due to the operation timeout, the session close failed, resulting in the engine not being killed.

### Affects Version(s)

1.4.0

### Kyuubi Server Log Output

```logtalk
2022-04-24 08:11:51,888 WARN  [KyuubiSessionManager-timeout-checker: Thread-8
7]: org.apache.kyuubi.operation.KyuubiOperationManager(60) - Operation Operat
ionHandle [type=UNKNOWN_OPERATION, identifier: e3d47534-072b-4970-952b-3929e7
107259] is timed-out and will be closed
```

```
2022-04-24 10:30:06,699 ERROR [KyuubiThriftBinaryFrontendServiceHandler-Pool:
 Thread-2060109]: org.apache.kyuubi.server.KyuubiThriftBinaryFrontendService(
72) - Error closing session: 
org.apache.kyuubi.KyuubiSQLException: Invalid OperationHandle [type=UNKNOWN_O
PERATION, identifier: e3d47534-072b-4970-952b-3929e7107259]
        at org.apache.kyuubi.KyuubiSQLException$.apply(KyuubiSQLException.sca
la:69)
        at org.apache.kyuubi.operation.OperationManager.removeOperation(Opera
tionManager.scala:88)
        at org.apache.kyuubi.operation.OperationManager.closeOperation(Operat
ionManager.scala:103)
        at org.apache.kyuubi.session.AbstractSession.$anonfun$closeOperation$
1(AbstractSession.scala:180)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java
:23)
        at org.apache.kyuubi.session.AbstractSession.withAcquireRelease(Abstr
actSession.scala:75)
        at org.apache.kyuubi.session.AbstractSession.closeOperation(AbstractS
ession.scala:178)
        at org.apache.kyuubi.session.KyuubiSessionImpl.close(KyuubiSessionImp
l.scala:143)
        at org.apache.kyuubi.session.SessionManager.closeSession(SessionManag
er.scala:93)
        at org.apache.kyuubi.service.AbstractBackendService.closeSession(Abst
ractBackendService.scala:49)
        at org.apache.kyuubi.service.ThriftBinaryFrontendService.CloseSession
(ThriftBinaryFrontendService.scala:227)
        at org.apache.hive.service.rpc.thrift.TCLIService$Processor$CloseSess
ion.getResult(TCLIService.java:1397)
        at org.apache.hive.service.rpc.thrift.TCLIService$Processor$CloseSess
ion.getResult(TCLIService.java:1382)
        at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:39)
        at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:39)
        at org.apache.kyuubi.service.authentication.HadoopThriftAuthBridgeSer
ver$TUGIAssumingProcessor.process(HadoopThriftAuthBridgeServer.scala:162)
        at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThre
adPoolServer.java:286)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecut
or.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecu
tor.java:617)
        at java.lang.Thread.run(Thread.java:745)
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

**Comment by cfmcgrady** [comment]
this should be fixed by https://github.com/apache/incubator-kyuubi/pull/1983

**Comment by wForget** [comment]
> this should be fixed by #1983

Thanks, it seems so.

> update lastAccessTime in getStatus

This change seems necessary, can you help me review #2452?
